packages/web-components/fast-components/src/accordion-item/accordion-item.definition.ts

```diff
@@ -11,21 +11,23 @@ export const fastAccordionItemDefinition: WebComponentDefinition = {
             attributes: [
                 {
                     name: "heading-level",
-                    description: "The heading level attribute",
+                    title: "Heading level",
+                    description: "The aria-level value (1-6) for the item heading",
                     type: DataType.number,
                     default: 2,
                     required: false,
                 },
                 {
                     name: "expanded",
-                    description: "The expanded attribute",
+                    title: "Expanded",
+                    description: "The expanded state of the item",
                     type: DataType.boolean,
                     default: false,
                     required: false,
                 },
                 {
                     name: "id",
-                    description: "The id attribute",
+                    description: "The HTML id attribute for the invoking element",
```

Comment:
```suggestion
                    title: "ID",
                    description: "The HTML id attribute for the invoking element",
```

Reply:
Does "invoking" add information? Can we remove it and assume this refers to the containing or logically grouping element?

Reply:
The id is specific to the button which invokes the item. I'm not sure this will benefit design no matter how we describe it, but it's accurate. IMO this one is more helpful for devs and I think that's okay. I think it'd be okay if a designer looked at the tooltip and said, "not important to me" as that is kind of accurate for this portion of the API :)